manifests/run.pp

```diff
@@ -432,17 +440,19 @@
         }
 
         if $running == false {
+          $running_stop_command = [$docker_command, 'stop', "--time=${stop_wait_time}", $sanitised_title]
```

Comment:
"--time=${stop_wait_time}"

This will need addressing.

Does the flag actually need the =? if not you could just have a clean break between --time and $stop_wait_time.

Reply:
Same as above.